libs/sdk-ui-kit/src/Header/generateHeaderHelpMenuItems.ts

```diff
@@ -0,0 +1,96 @@
+// (C) 2020-2021 GoodData Corporation
+
+import { IFeatureFlags } from "@gooddata/goodstrap/lib/@next/Header/types";
```

Comment:
same as in previous file

Reply:
Thanks I used it instead of using from goodstrap